app-emulation/docker-runc/docker-runc-1.0.0_rc91.ebuild

```diff
@@ -11,8 +11,8 @@ COREOS_GO_VERSION="go1.13"
 # Update the patch number when this commit is changed (i.e. the _p in the ebuild).
 # The patch version is arbitrarily the number of commits since the tag version
 # specified in the ebuild name. For example:
-# $ git log --oneline v1.0.0-rc90..${COMMIT_ID} | wc -l
-COMMIT_ID="dc9208a3303feef5b3839f4323d9beb36df0a9dd"
+# $ git log --oneline v1.0.0-rc91..${COMMIT_ID} | wc -l
+COMMIT_ID="42fff598cfbc9bd49f1d9d78e1e9d78b0921ddb0"
```

Comment:
Is this commit ID wrong? https://github.com/opencontainers/runc/commit/42fff598cfbc9bd49f1d9d78e1e9d78b0921ddb0 gives 404. In the releases page, rc91 is at 24a3cf88a7ae5f4995f6750654c0e2ca61ef4bb2.

Reply:
To expand on it - I've got a build failure with the following error:
```
--2020-08-04 19:52:30--  https://github.com/opencontainers/runc/archive/42fff598cfbc9bd49f1d9d78e1e9d78b0921ddb0.tar.gz
Resolving github.com... 140.82.118.3
Connecting to github.com|140.82.118.3|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://codeload.github.com/opencontainers/runc/tar.gz/master [following]
--2020-08-04 19:52:31--  https://codeload.github.com/opencontainers/runc/tar.gz/master
…
…
…
!!! Fetched file: docker-runc-1.0.0_rc91.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      2061612
!!! Expected: 1939849
Refetching... File renamed to '/mnt/host/source/.cache/distfiles/docker-runc-1.0.0_rc91.tar.gz._checksum_failure_.mL04zx'

!!! Couldn't download 'docker-runc-1.0.0_rc91.tar.gz'. Aborting.
```

Notice that github does some redirect to the archive of the master branch.

Reply:
> Is this commit ID wrong? https://github.com/opencontainers/runc/commit/42fff598cfbc9bd49f1d9d78e1e9d78b0921ddb0 gives 404. In the releases page, rc91 is at 24a3cf88a7ae5f4995f6750654c0e2ca61ef4bb2.

Both `42fff598cfbc9bd49f1d9d78e1e9d78b0921ddb0` and `24a3cf88a7ae5f4995f6750654c0e2ca61ef4bb2` are correct. Both commits point to `v1.0.0-rc91`. However, the former points to a commit with GPG signature, while the latter represents a commit without GPG signature.

Github actions [create](
https://github.com/flatcar-linux/coreos-overlay/blob/d21e113b9365989291902d8fe672897bd34a1ead/.github/workflows/runc-releases-main.yml#L21) the commit by running `git rev-parse refs/tags/v1.0.0-rc91`. That's why we see the result commit `42fff598cfbc9bd49f1d9d78e1e9d78b0921ddb0` with GPG sig. If we would want to make the commit point to the commit without GPG sig, then we need to run instead `git rev-parse refs/tags/v1.0.0-rc91^{commit}`.

```
$ git clone https://github.com/opencontainers/runc
$ cd runc
$ git rev-parse refs/tags/v1.0.0-rc91
42fff598cfbc9bd49f1d9d78e1e9d78b0921ddb0
$ git rev-parse refs/tags/v1.0.0-rc91^{commit}
24a3cf88a7ae5f4995f6750654c0e2ca61ef4bb2
```

> !!! Fetched file: docker-runc-1.0.0_rc91.tar.gz VERIFY FAILED!
> !!! Reason: Filesize does not match recorded size
> !!! Got:      2061612
> !!! Expected: 1939849
> Refetching... File renamed to '/mnt/host/source/.cache/distfiles/docker-runc-1.0.0_rc91.tar.gz._checksum_failure_.mL04zx'

Both URLs are valid, and should work fine, https://github.com/opencontainers/runc/archive/24a3cf88a7ae5f4995f6750654c0e2ca61ef4bb2.tar.gz and https://github.com/opencontainers/runc/archive/42fff598cfbc9bd49f1d9d78e1e9d78b0921ddb0.tar.gz .

The checksum error above is another issue.
It happens probably because some garbage tarballs remains under `~/trunk/.cache/distfiles/`.
Please try cleaning up all files like `~/trunk/.cache/distfiles/docker-runc-1.0.0_rc91.tar.gz` and reinstalling `docker-runc`.